codalab/worker/download_util.py

```diff
@@ -131,3 +145,84 @@ def _compute_target_info(path, depth):
     if result is None:
         raise PathException()
     return result
+
+
+def _compute_target_info_beam(path, depth):
+    # TODO (Ashwin): handle depth.
+    bundle_uuid, zip_path, zip_subpath = parse_azure_url(path)
+    if zip_subpath == "/images/":
```

Comment:
What's special about `/images/`?  Please put more comments.

Reply:
That was also for testing -- updated now.